Dockerfile

```diff
@@ -46,13 +46,31 @@ RUN set -ex && \
     $sudo rm -rf /tmp/* && \
     yum clean all
 
+# SSL dependency gets baked into Python binary so do this first
+RUN yum -y update && \
+  yum install -y perl-core pcre-devel && \
+  wget https://ftp.openssl.org/source/openssl-1.1.1k.tar.gz && \
+  tar -xzvf openssl-1.1.1k.tar.gz && \
+  cd openssl-1.1.1k && \
+  ./config shared --prefix=/usr/local/ssl --openssldir=/usr/local/ssl && \
+  make && \
+  make install && cd .. && \
+  $sudo yum -y remove perl-core pcre-devel && \
+  ldconfig
+
```

Comment:
could you add the yum cleanup steps here.

Reply:
yep, good call!